tests/__fixtures__/bags/bag.fixtures.ts

```diff
@@ -93,6 +104,14 @@ export const mockErrorState = {
         },
       },
     },
+    bagOperations: {
+      error: {
+        [mockBagOperationId]: { message: 'An unexpected error occurred' },
```

Comment:
Prefer using an `Error` instance cast to `BlackoutError` type, like this:

```suggestion
        [mockBagOperationId]:  new Error('An unexpected error occurred') as BlackoutError,
```

I know there are other mocks defined in the same way that are not from your PR, but could you normalize them please?

Reply:
Done!